### PR TITLE
feat(oas): implement security declaration type

### PR DESCRIPTION
**Pull request by daniel-white**
implements the security declaration type from the types packages
fixes a bunch of broken tests

### CODE REVIEW

File: src/oas/operation.ts [hunk 1]
```diff
@@ -95,5 +96,7 @@ export const transformOasOperation: TranslateFunction<
       },
       isString,
     ),
+
+    securityDeclarationType: translateToSecurityDeclarationType(operation),
```
Comment (daniel-white) [inline src/oas/operation.ts:100]: relevant change

File: src/oas/transformers/securities.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { HttpOperationSecurityDeclarationTypes, IHttpOperation } from '@stoplight/types';
+import type { OperationObject } from 'openapi3-ts/src/model/OpenApi';
+import type { Operation } from 'swagger-schema-official';
+
+export function translateToSecurityDeclarationType({
```
Comment (daniel-white) [inline src/oas/transformers/securities.ts:5]: the actual implemented logic

File: src/postman/__tests__/service.test.ts [hunk 3]
```diff
@@ -34,6 +34,7 @@ describe('transformPostmanCollectionService()', () => {
               key: 'http-0',
               scheme: 'basic',
               type: 'http',
+              extensions: {},
```
Comment (brendarearden) [inline src/postman/__tests__/service.test.ts:37]: there are showing up empty now because extensions is always required?

**Review by brendarearden — APPROVED** [review]
lgtm! just had one question but its not a change request.

File: src/postman/__tests__/service.test.ts (hunk 3 above)
Comment (daniel-white) [reply src/postman/__tests__/service.test.ts:37]: yea - someone made a breaking change in stoplight/types w/o upgrading here

**Pull request merged**